backend/src/main/java/org/sefglobal/core/config/SecurityConfig.java

```diff
@@ -43,6 +43,7 @@ protected void configure(HttpSecurity http) throws Exception {
                 .csrf().disable()
                 .authorizeRequests()
                     .antMatchers("/link/*").permitAll()
+                    .antMatchers("/academix/*").permitAll()
```

Comment:
If we do it this way, the add APIs also exposed to public.

Reply:
Let's name those APIs as `academix/admin` or something as we discussed.